geosnap/data/data.py

```diff
@@ -90,6 +83,26 @@ def blocks_2000(self, states=None, convert=True):
             stored as well-known binary on the 'wkb' column.
 
         """
+
+        try:  # if any of these aren't found, stream them insteead
```

Comment:
Are these two data sets only/mainly used for harmonization? If so, maybe adopt some strategy so that users who do not intend to employ harmonization methods do not need to pay attention to/download these large two data sets? Could be:
* providing more explicit warning info? 
* using the checking in the harmonization module - only when the two data sets are needed?

Reply:
the block geometries are used in the `from_lodes` constructor (lehd data), and with this restructuring, the warnings shouldn't get triggered until a user calls that method or tries to use blocks from the data store. So unlike the other census data that raises a warning on import because its used in lots of other places in the package, the only time you see the block warnings is if you try to actually use blocks. Does that accomplish what you'd like?

I'm definitely up for a better warning, what would you suggest?

Reply:
to be honest, i'm open to removing the warnings altogether since the streaming seems to work really well in mose cases and we added the check in case streaming fails

Reply:
I missed that the checkings are within functions. Looks good!

Reply:
I think the warnings are very helpful - actually, streaming never worked when I was using the WIFI at home - always timed out. Local storing is super useful, and the warnings inform me of how to store the data locally.

Reply:
good to know! then lets definitely keep them. Maybe we should also add a reference to the appropriate `store_` function in the [stream failure](https://github.com/spatialucr/geosnap/blob/5b2715d947aaea025f158fd8d1474ea3d36fc483/geosnap/data/data.py#L62) warning?

Reply:
Does not the [warn][https://github.com/spatialucr/geosnap/blob/5b2715d947aaea025f158fd8d1474ea3d36fc483/geosnap/data/data.py#L47] already provide the reference?

Reply:
thinking through it, i guess this is what i meant in the prior comment. 

currently, we default to local data, raise a warning with download instructions if its not found, then try streaming the data and raise a generic warning if that fails.

instead I'm, proposing we default to local data, and fallback silently to streaming if its not found, then raise an informative user error with download instructions if that fails. We'd put the download instructions in the docs instead

or, we could just warn each time and include reference the `store_` functions both times

Reply:
The current strategy looks good to me! Users can opt for silencing all warnings if they do not feel like getting all the warnings.